### DISCUSSION TITLE
Respect skim environment variables

### DISCUSSION THREAD

**Discussion by mk12** [arxanas/git-branchless#241] [Ideas]
Skim's default colors do not play well with my base16 terminal setup.

Skim has `SKIM_DEFAULT_OPTIONS` and `SKIM_DEFAULT_COMMAND`. I was hoping to just copy my fzf equivalents into those. However, only the skim binary reads those, not the library. There's an issue about it here: https://github.com/lotabout/skim/issues/390.

Once skim provides a helper to read those variables and create `SkimOptions`, I think git-branchless should use that. Or maybe in the meantime it could duplicate some of the logic to do that, or use its own environment variables to configure things like color.

**Comment by arxanas** [comment]
Thanks for the suggestion. I agree that it would be good to configure the Skim viewer. It's not a priority for me right now, but I'm happy to accept a PR to do this.
